Quaver.API/Maps/Qua.cs

```diff
@@ -362,6 +362,7 @@ public string Serialize()
             TimingPoints = originalTimingPoints;
             HitObjects = originalHitObjects;
             SoundEffects = originalSoundEffects;
+            Bookmarks = new List<BookmarkInfo>();
```

Comment:
This needs to copy the original bookmarks just like the other ones. 

You're clearing it here, so it will always get rid of any bookmarks the user places.